x-pack/plugins/siem/public/components/load_more_table/index.tsx

```diff
@@ -189,17 +197,23 @@ export class LoadMoreTable<T, U, V, W, X, Y, Z, AA, AB> extends React.PureCompon
               <LoadingPanel
                 height="100%"
                 width="100%"
-                text={`${i18n.LOADING} ${loadingTitle ? loadingTitle : title}`}
+                text={`${i18n.LOADING} ${loadingTitle ? loadingTitle : headerTitle}`}
                 position="absolute"
                 zIndex={3}
                 data-test-subj="LoadingPanelLoadMoreTable"
               />
             </>
           )}
-          <EuiTitle size="s">
-            <>{title}</>
-          </EuiTitle>
-          <EuiBasicTable
+
+          <HeaderPanel
+            subtitle={`Showing: ${headerCount} ${headerUnit}`}
```

Comment:
Is the word `Showing:` suppose to be an i18n string?

Reply:
Bah, I always miss these. Will fix. Thanks!